components/chassis.py

```diff
@@ -294,7 +294,7 @@ def execute(self) -> None:
 
         self.update_odometry()
         # rotation2d and translation2d have mul but not div
-        real_chassis_speeds = self.kinematics.toChassisSpeeds(
+        self.kinematics.toChassisSpeeds(
```

Comment:
This looks like a pure function whose result is unused...

Reply:
yes, douglas and james fixed this in their auto move branch, we'll get it merged soon

Reply:
Okay, so I should just leave this call here as is then?

Reply:
yeah